README.md

```diff
@@ -11,24 +11,48 @@ Image Template to build custom notebook images for JupyterHub.
 ## How to build the image
 
 ### Locally
+
+#### Prerequisites
+
+* **s2i**: We assume majority of users will be using linux and recommend using the the instructions 
+[here](https://github.com/openshift/source-to-image#for-linux) to install s2i if needed.    
+
+* **Podman**: Please follow the instructions [here](https://github.com/openshift/source-to-image#for-linux) for your 
+linux distribution. 
+
+* **Docker**: If you are using Fedora please use these [instructions](https://docs.docker.com/engine/install/fedora/) 
+to install docker. If you are on RHEL8 you can run into issues with docker, please follow the additional documentation
+that can be found [here](docs/RHEL8_docker_install.md).   
+
+#### Build
+
   You can build the image locally using s2i, by executing the following command
   in the root of your repository. <br>
-    `s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest`
+````
+$ s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest
+````
+Be sure to replace `MY-CUSTOM-IMAGE` above with the name you intend to use for your image. 
 
 You might need to update your deployment of JupyterHub for the newly built/imported image to
 be available for use.
 
 # Running the image Locally
+
 After building the image, you can run it locally using the following command: <br>
-`podman run -p 8080:8080 MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080`
+````
+$ podman run -p 8080:8080 docker-daemon:MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080
+````
+Since the s2i notebook builder relies on docker, and we prefer to use podman, make sure to append your image name with
+`docker-daemon:` so that podman checks the correct local registry.  
 
 ## How to add the image to Open Data Hub/JupyterHub
 
 * Once you have pushed your image to an image registry that is accessible by your JupyterHub instance,
 you can use the `ImageStream` template in this repository at `openshift/oc-imagestream.yaml`.
 
 
-```yaml
+```
+yaml
```

Comment:
the `yaml` should go on the same line as the ` ``` `

Reply:
This did not render correctly with `yaml` on the same line in my pycharm editor. But looks like it works correctly on github : ) 

Change made